src/microbe_stage/editor/LightConfigurationPanel.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   The microbe editor's light configuration top panel
+/// </summary>
+public partial class LightConfigurationPanel : PanelContainer
+{
+#pragma warning disable CA2213
+
+    [Export]
+    private Button dayButton = null!;
+
+    [Export]
+    private Button nightButton = null!;
+
+    [Export]
+    private Button averageLightButton = null!;
+
+    [Export]
+    private Button currentLightButton = null!;
+
+#pragma warning disable CA2213
```

Comment:
```suggestion
#pragma warning restore CA2213
```

I think that's a typo.